### PR TITLE
Doc updates: build and running on YARN

### PR DESCRIPTION
**Pull request by felixcheung**
@shivaram Thoughts?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,6 +150,18 @@ YARN_CONF_DIR=/root/ephemeral-hdfs/conf/ MASTER=yarn-client ./sparkR
 YARN_CONF_DIR=/root/ephemeral-hdfs/conf/ ./sparkR examples/pi.R yarn-client
 ```
 
+### Using sparkR-submit
+sparkR-submit is a script introduced to faciliate submission of SparkR jobs to a YARN cluster.
+It supports the same commandline parameters as [spark-submit](http://spark.apache.org/docs/latest/running-on-yarn.html). SPARK_HOME, YARN_HOME, and JAVA_HOME must be defined.
```
Comment (shivaram) [inline README.md]: Actually we should link to http://spark.apache.org/docs/latest/submitting-applications.html -- And it'll also be good to work in the fact that sparkR-submit can be used to submit jobs to any cluster manager (similar to spark-submit)

**Comment by shivaram** [comment]
Thanks @felixcheung - This is very useful. I just had one comment about sparkR-submit

**Comment by felixcheung** [comment]
Updated for add more info on sparkR-submit. Thanks!

File: README.md [hunk 2]
```diff
@@ -121,6 +150,23 @@ YARN_CONF_DIR=/root/ephemeral-hdfs/conf/ MASTER=yarn-client ./sparkR
 YARN_CONF_DIR=/root/ephemeral-hdfs/conf/ ./sparkR examples/pi.R yarn-client
 ```
 
+## Running on a cluster using sparkR-submit
+
+sparkR-submit is a script introduced to faciliate submission of SparkR jobs to a Spark supported cluster (eg. Standalone, Mesos, YARN).
+It supports the same commandline parameters as [spark-submit](http://spark.apache.org/docs/latest/submitting-applications.html). SPARK_HOME and JAVA_HOME must be defined.
+
+On YARN, YARN_HOME must be defined. Currently, SparkR only supports [yarn-client](http://spark.apache.org/docs/latest/running-on-yarn.html) mode.
```
Comment (shivaram) [inline README.md]: Minor correction: With sparkR-submit its not just yarn-client, but you can also use yarn-cluster for running batch scripts (yarn-cluster doesn't work with the shell).

**Comment by felixcheung** [comment]
@shivaram - I've made some updates, could you take a look?

**Comment by shivaram** [comment]
Thanks @felixcheung - LGTM. Merging this.

**Pull request merged**